examples/bri-3/src/bri/zeroKnowledgeProof/api/anchorHash.controller.ts

```diff
@@ -1,24 +1,12 @@
 import { Body, Controller, Post } from '@nestjs/common';
 import { CommandBus } from '@nestjs/cqrs';
-import { CreateAnchorHashCommand } from '../capabilities/createAnchorHash/createAnchorHash.command';
 import { VerifyAnchorHashCommand } from '../capabilities/verifyAnchorHash/verifyAnchorHash.command';
-import { CreateAnchorHashDto } from './dtos/request/createAnchorHash.dto';
 import { VerifyAnchorHashDto } from './dtos/request/verifyAnchorHash.dto';
-import { AnchorHashDto } from './dtos/response/anchorHash.dto';
```

Comment:
is this dto used at some other place?

Reply:
AnchorHashDto is also used in [anchorHash.profile.ts](https://github.com/eea-oasis/baseline/blob/main/examples/bri-3/src/bri/zeroKnowledgeProof/anchorHash.profile.ts)

Reply:
I believe it can also be removed as we used it only when creating a new anchor hash

Reply:
@ognjenkurtic Do you mean remove AnchorHashDto completely as well? Or just remove AnchorHashDto in this controller?

Also I'm wondering do you need update this [doc](https://docs.google.com/document/d/1mkO5oou0TWk6kVHbTB01dyj7wuNGTfR9w6PAxSgXdGI/edit#) to reflect removing createAnchorHash? Thanks!

Reply:
Yes, you can remove AnchorHashDto completely from the codebase.The document you referenced is now out of date and the relevant one is the [SRI use-case](https://docs.google.com/document/d/14W2Pr00K8SBPPO7-aQiDP73uMAEwd6rwK7Tvtvfxeng/edit?usp=sharing) doc. We ll be updating all other documentation as soon as the discussion in the SRI use-case doc is finalized.

Reply:
Done. Please let me know if there is anything else I can help with.